README.md

```diff
@@ -45,12 +45,12 @@ Datasets Comparison
 Usage: spark-submit [spark options] --class za.co.absa.hermes.datasetComparison.DatasetComparisonJob hermes.jar [options]
 
   -f, --raw-format <value> format of the raw data (csv, xml, parquet,fixed-width, etc.)
-  --row-tag <value>        use the specific row tag instead of 'ROW' for XML format
+  --rowTag <value>        use the specific row tag instead of 'ROW' for XML format
```

Comment:
Why camelCase?

Reply:
The official option from spark-xml is camel case. This is so I don't need to go through every possible option offered by different formats, parse is, etc. and this way I leverage the code that is already there.